### PR TITLE
DTFS2-5394 - GEF Bug - Comments name not showing

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made: 

* Added full comment object to return-to-maker and submit-to-ukef
* Fixed gef tests
* Added to relevant e2e tests to ensure name shows up

### CODE REVIEW

File: portal-api/package-lock.json [hunk 1]
```diff
@@ -18,6 +21,8 @@
         "dotenv": "16.0.0",
         "express": "4.17.2",
         "filesize": "^8.0.7",
+        "graphql": "^15.8.0",
+        "graphql-middleware": "^4.0.2",
```
Comment (ttbarnes) [inline portal-api/package-lock.json]: Is this change needed? Thought I removed graphql/apollo from portal api

Comment (Zainzzkk) [reply portal-api/package-lock.json]: currently in main->portal-api->package.json .  I did a npm install

**Review by IgnatG — APPROVED** [review]
